### DISCUSSION TITLE
hello, Are there events about full screen?

### DISCUSSION THREAD

**Discussion by rookiewxy** [cookpete/react-player#1795] [Q&A]
Be sure to [search for your issue](https://github.com/CookPete/react-player/issues) before opening a new one.

#### Steps to Reproduce

1.
1.
1.

#### Environment
- URL attempting to play: 
- Browser: 
- Operating system: 
- jsFiddle example: https://jsfiddle.net/sv5x3ug1

**Comment by nzhiti** [comment]
Reactplayer is using [screenfull ](https://github.com/sindresorhus/screenfull) library to handle fullscreen.

`if (screenfull.isEnabled) {
	screenfull.on('change', () => {
		console.log('Am I fullscreen?', screenfull.isFullscreen ? 'Yes' : 'No');
	});
}`

You can also try to use the full screen browser API : https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenchange_event

`addEventListener("fullscreenchange", (event) => {});`
